config/nginx.conf

```diff
@@ -0,0 +1,26 @@
+upstream gb-botler {
+  server unix:///var/www/gb-botler/shared/sockets/puma.sock;
+}
+
+server {
+  listen 8080;
```

Comment:
I tried, but that breaks redirects. I guess that's because the value of `$server_port` is equal to the value passed to `listen`.